qeb_thamos_advise.py

```diff
@@ -97,37 +96,53 @@ def qeb_hwt_thamos_advise() -> None:
         trigger_finished_webhook(exception_message=exception_message, has_error=True, error_type="MissingThothYamlFile")
         return
 
+    # Fetch recommedation type
     try:
-        analysis_id = advise_here(
-            nowait=True,
-            github_event_type=Configuration._GITHUB_EVENT_TYPE,
-            github_check_run_id=Configuration._GITHUB_CHECK_RUN_ID,
-            github_installation_id=Configuration._GITHUB_INSTALLATION_ID,
-            github_base_repo_url=Configuration._GITHUB_BASE_REPO_URL,
-            origin=Configuration._ORIGIN,
-            source_type=ThothAdviserIntegrationEnum.GITHUB_APP,
-        )
-        _LOGGER.info("Successfully submitted thamos advise call: %r", analysis_id)
-    except Exception as exception:
-        if isinstance(
-            exception,
-            (
-                NoRuntimeEnvironmentError,
-                NoRequirementsFormatError,
-                FileNotFoundError,
-                FileLoadError,
-                KeyError,
-                ValueError,
-                AttributeError,
-            ),
-        ):
-            _LOGGER.debug(exception)
-            exception_message = str(exception)
+        thoth_yaml_runtime_env = thoth_yaml_config.content.get("runtime_environments")
+        recommendation_type = thoth_yaml_runtime_env[0].get("recommendation_type") if thoth_yaml_config else "latest"
+
+        requirements_format = thoth_yaml_config.requirements_format
+        if requirements_format == "pipenv":
+            project = Project.from_files(without_pipfile_lock=not os.path.exists("Pipfile.lock"))
+        elif requirements_format in ("pip", "pip-tools", "pip-compile"):
+            project = Project.from_pip_compile_files(allow_without_lock=True)
         else:
-            _LOGGER.debug(json.loads(exception.body)["error"])  # type: ignore
-            exception_message = json.loads(exception.body)["error"]  # type: ignore
+            raise ValueError(f"Unknown configuration option for requirements format: {requirements_format!r}")
 
+        pipfile = project.pipfile.to_string()
+        pipfile_lock_str = project.pipfile_lock.to_string() if project.pipfile_lock else ""
+        application_stack = PythonStack(
```

Comment:
the input to AdviserTriggerMessage is not PythoStack type but dict: https://github.com/thoth-station/user-api/blob/13e877a51fcff47336a4a319c2b486f268ff54d3/thoth/user_api/api_v1.py#L297 we use the method used in User-API here we do not talk to user-API

Reply:
I am passing the type dict now.